EvenImageDivider.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+class EvenImageDivider:
+	def __init__(self, image,n):
+		self.image = image
+		self.n = n
+
+	def halvenAxis(self,axisLength,n):
+		quotient = axisLength/n
+		i=1
+		remainder = axisLength % n
+		result=[]
+		result.append(quotient + remainder)
+		for x in xrange(0,n-1):
+			result.append((i+1)*quotient+remainder)
+			i+=1
+		return [0] + result
+
+	def divider(self,sector):
+		rows = len(self.image)
+		cols= len(self.image[0])
+		rowSplit=self.halvenAxis(rows,2)
+		colSplit=self.halvenAxis(cols,self.n/2)
+		result=[]
+		counter=1
+		for i in xrange(0,len(rowSplit)-1):
```

Comment:
@medo:
The loop starting at line#14 calculates the boundaries of the images. 

The loop starting at line#26 is needed because when extracting n-sectors of an image, they're not necessarly the same size. So the approach with putting all (let's say quadrants in this case with n=4) in one numpy array won't work, because essentially they'll have the same size. So the loop returns the needed sector.